### ISSUE TITLE
Change C2DMessage permission specification?

### ISSUE THREAD

**Issue by avranju** [MSOpenTech/msopentech-tools-for-intellij#176]
The C2DMessage permission added has the following format:
```
<permission android:name="com.microsoftopentechnologies.intellij.permission.C2D_MESSAGE" android:protectionLevel="signature”/>
```
While not technically incorrect, I’m not sure if this would cause issues if multiple apps are installed on a device with the same permission (if someone didn’t think to change it).  I’d recommend changing this to pull the package name of the application you’re adding the activity to so it would be in this format:
```
<permission android:name=“<my-apps-package>.permission.C2D_MESSAGE" android:protectionLevel="signature”/>
```
This would also require changing the **uses-permission** and the **intent-filter**’s **category**.

**Comment by leotilli** [comment]
Fixed on Jan 20th (8cfbd72)

And merged on Jan 28th (#113)

**Comment by kasturiswain** [comment]
This issue is fixed and functionality is working as expected in the latest build.
